### PR TITLE
Up2023.10

### PR DESCRIPTION
**Pull request by montjoie**
Upgrade to 2023.10 and add some security by closing some unnecessary ports

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -344,6 +344,7 @@ boards:
       devpath: the UDEV devpath to this uart for UART without serial number
       interfacenum:	(optional) The interfacenumber of the serial. (Used with two serial in one device)
       use_ser2net: 	True/False (Deprecated, ser2net is the default uart handler)
+      worker:          (optional) an host/IP where ser2net is running
```
Comment (patersonc) [inline README.md:348]: Would this be better as `ser2net_worker` as it's specifically for ser2net, like the other variables below?

Obviously `lavalab-gen.py` would need updating accordingly.

**Review by patersonc — COMMENTED** [review]
Thank you montjoie!

File: README.md (hunk 1 above)
Comment (montjoie) [reply README.md:348]: It could be used for other purpose, with labgrid ser2net will be not used, but this information remains the same.

Comment (patersonc) [reply README.md:348]: Ah okay :)

**Pull request merged**